src/main/resources/build.template.properties

```diff
@@ -19,6 +19,12 @@ app.region=us-central1
 # e.g. app.version = 8-0-0
 app.version = 8-0-0
 
+# This is the base URL for production environment.
+# It should not contain trailing slashes ('/').
+# This is mainly used as the prefix for links sent in outbound emails.
+# If using GAE, the default domain name will be https://{app_id}.appspot.com
+app.production.url = https\://teammates-john.appspot.com
```

Comment:
Perhaps we just comment this away and leave a `app.production.url=` to start with so that any first timer will not be surprised by teammates-john